### DISCUSSION TITLE
Stable Diffusion 3 announced - waitinglist is open

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#2336] [Ideas]
StabilityAI has opened the waiting list for Stable Diffusion 3.

Personally i'm super excited to get access to it and try out all of the new improvements such as multi subject rendering and better overall quality!

Check it out and sign up for the waiting list here: https://stability.ai/news/stable-diffusion-3

**Comment by quasar136a8** [comment]
In cases of providing access, how can we use the new version in fooocus?

**Reply by mashb1t** [reply]
We'll see when getting getting access to it.

**Comment by blckbx** [comment]
Research paper released: 

https://stability.ai/news/stable-diffusion-3-research-paper
https://stabilityai-public-packages.s3.us-west-2.amazonaws.com/Stable+Diffusion+3+Paper.pdf

**Reply by damadorPL** [reply]
![Screenshot_2024-06-12-15-59-47-176_com.google.android.gm.jpg](https://github.com/lllyasviel/Fooocus/assets/16166558/3d0371fa-651a-4ad2-b3bf-c0e4a5e6e1fe)

Just released

**Reply by blckbx** [reply]
https://huggingface.co/stabilityai/stable-diffusion-3-medium

**Comment by quasar136a8** [comment]
Guys how to use it with focus (what to download) what and were to copy it in focus. Ths

**Reply by quasar136a8** [reply]
<img width="643" alt="Screenshot 2024-06-12 171226" src="https://github.com/lllyasviel/Fooocus/assets/111929719/e090ad88-5890-4dee-9dec-6c7e612f4663">

**Reply by xhoxye** [reply]
We have prepared three packaging variants of the SD3 Medium model, each equipped with the same set of MMDiT & VAE weights, for user convenience.

sd3_medium.safetensors includes the MMDiT and VAE weights but does not include any text encoders.
sd3_medium_incl_clips_t5xxlfp8.safetensors contains all necessary weights, including fp8 version of the T5XXL text encoder, offering a balance between quality and resource requirements.
sd3_medium_incl_clips.safetensors includes all necessary weights except for the T5XXL text encoder. It requires minimal resources, but the model's performance will differ without the T5XXL text encoder.
The text_encoders folder contains three text encoders and their original model card links for user convenience. All components within the text_encoders folder (and their equivalents embedded in other packings) are subject to their respective original licenses.
The example_workfows folder contains example comfy workflows.

**Comment by quasar136a8** [comment]
we copy it to models/checkpoints - correct?

**Reply by ILikeToasters** [reply]
Fooocus does not have support for it yet.  And if it did you might be disappointed since the initial results are not great as expected.

**Reply by quasar136a8** [reply]
<img width="692" alt="image" src="https://github.com/lllyasviel/Fooocus/assets/111929719/ec74305f-dfe9-43d4-bad0-95506c89c608">
After first running with prompt model "sd3_medium_incl_clips_t5xxlfp8.safetensors" stop working (and not showing in the list)

**Reply by IPv6** [reply]
> After first running with prompt model "sd3_medium_incl_clips_t5xxlfp8.safetensors" stop working (and not showing in the list)

Yep, because inners of the format are not compatible with SDXL/SD15 loras
SD3 is different, it`s not as simple as "replace a file"

**Comment by mashb1t** [comment]
Fooocus does not (yet) support SD3.
Please find more information in 
- https://github.com/lllyasviel/Fooocus/discussions/2773#discussioncomment-9643676
- https://github.com/mashb1t/Fooocus/discussions/31#discussioncomment-9474519
- https://github.com/lllyasviel/Fooocus/pull/3084

**Reply by AnalyticsXXI** [reply]
> Fooocus does not (yet) support SD3. 

Do you have plans to add it in the near future? Like, within a week, for example. Or no any specific plans yet?

**Reply by mashb1t** [reply]
@AnalyticsXXI your support for https://github.com/lllyasviel/Fooocus/pull/3084 is welcome if you want to contribute.
I'm currently working on https://github.com/lllyasviel/Fooocus/discussions/3122

**Reply by AnalyticsXXI** [reply]
> your support for #3084 is welcome if you want to contribute

I couldn't even figure out, what's this ticket about, so don't count on me :)

As for the answer to my question, I read it as "We are currently too busy to make plans", is it somewhat correct interpretation?

**Reply by mashb1t** [reply]
@AnalyticsXXI i can only speak for myself but yes. I've started with the patching, then encountered some issues and asked for support in https://github.com/lllyasviel/Fooocus/discussions/2773#discussioncomment-9643676, so that's that. I strongly assume that currently no tool supports SD3 (a1111, vladmandic, comfy, fooocus, invoke, etc.).

**Reply by mashb1t** [reply]
update (sorry, was mistaken): you can find comfy workflows in https://huggingface.co/stabilityai/stable-diffusion-3-medium/tree/main/comfy_example_workflows

**Reply by CantainPike** [reply]
Works with StableSwarm beta too, anyone who does not like ComfyUI
https://github.com/Stability-AI/StableSwarmUI
they have pretty active discord channel too
